tests/unittests/config/test_cc_ubuntu_advantage.py

```diff
@@ -251,13 +269,21 @@ def test_configure_ua_attach_on_service_enabled(self, m_subp, caplog):
 
         def fake_subp(cmd, capture=None, rcs=None, logstring=None):
             fail_cmds = [
-                ["ua", "enable", "--assume-yes", svc] for svc in ["livepatch"]
+                ["ua", "enable", "--assume-yes", "--format", "json", svc]
+                for svc in ["livepatch"]
             ]
             if cmd in fail_cmds and capture:
-                svc = cmd[-1]
-                raise subp.ProcessExecutionError(
-                    'Service "{}" is already enabled.'.format(svc)
-                )
+                response = {
+                    "errors": [
```

Comment:
I think we need to set the exit code 1 on this response and assert handling that failure path